terraform/variables.tf

```diff
@@ -79,6 +79,10 @@ variable "private_subnet_id" {
   description = "Identifier for private AWS subnet"
 }
 
+variable "peers" {
+  description = "List of initial Supervisor peers"
+}
```

Comment:
Based on this description I was expecting `peers` to be an actual list, but looking over at the [companion PR](https://github.com/habitat-sh/cloud-environments/pull/47), it seems that this is actually a string. I suppose this answers my earlier question about what happens if this "list" is empty, because without a default defined here, it has to be non-empty (there's the pathological case of an empty string, though, but we'd probably have much bigger issues if we got down to here with that).

Still looking at the other PR, but I'd recommend at least tweaking this description to reflect that this is already going to be a properly-formatted, non-empty string that gets created over in the cloud-environments repo. 

It may be worth refactoring eventually, such that that string formatting actually happens over here, just in case any community members happen to be using this Terraform code (either directly, or as inspiration) for their own deployments.

EDIT: OK, I've read through the other PR, so I get why this is just a string here - the logic for creating this peer list is also needed for some of our private instances over there. I feel like I want to refactor that, but I'm not sure what a good way to do that would be 😦 . I'd still like to tweak this `description` field, but other than that, let's keep going with the current implementation.